src/context.rs

```diff
@@ -536,21 +553,24 @@ impl Context {
     /// Signal an ongoing process to stop.
     pub async fn stop_ongoing(&self) {
```

Comment:
Yeah, I agree that `cancel_ongoing()` would be a nicer name.  I guess I could change that as well.  I didn't touch it because it is a pub API though and I feel like if we try and rename it maybe it should also be renamed in the FFI and JSON-RPC APIs and now we're breaking all clients.  That seemed like a bit too much work for a slightly unfortunate name.